package.json

```diff
@@ -24,28 +24,33 @@
     "pretest": "npm run build",
     "test": "mocha",
     "posttest": "npm run lint",
-    "prepublish": "npm run test"
+    "prepublishOnly": "npm run test"
   },
   "dependencies": {
-    "compare-versions": "2.0.1",
-    "object-assign": "^4.0.1",
-    "rollup-pluginutils": "^1.3.1",
-    "tippex": "^2.1.1",
-    "typescript": "^1.8.9"
+    "rollup-pluginutils": "^2.0.1",
+    "tippex": "^3.0.0"
   },
   "devDependencies": {
-    "buble": "^0.13.1",
-    "eslint": "^2.13.1",
-    "mocha": "^3.0.0",
+    "@types/node": "9.6.7",
+    "buble": "^0.19.3",
+    "eslint": "^4.19.1",
+    "mocha": "^5.1.1",
     "rimraf": "^2.5.4",
-    "rollup": "^0.49.3",
-    "rollup-plugin-buble": "^0.13.0"
+    "rollup": "^0.58.2",
+    "rollup-plugin-buble": "^0.19.2",
+    "rollup-plugin-node-resolve": "^3.3.0",
+    "tslib": "^1.9.0",
+    "typescript": "^2.8.3"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/rollup/rollup-plugin-typescript.git"
   },
   "bugs": {
     "url": "https://github.com/rollup/rollup-plugin-typescript/issues"
+  },
+  "peerDependencies": {
+    "tslib": "^1.9.0",
+    "typescript": "1.x || 2.x"
```

Comment:
Perhaps I am misunderstanding, but based on [tslib's installation instructions](https://github.com/Microsoft/tslib#installing) (reproduced below) I believe that `"tslib": "^1.9.0"` would require `"typescript": "^2.3.3"`.

> ```
> # TypeScript 2.3.3 or later
> npm install --save tslib
>
> # TypeScript 2.3.2 or earlier
> npm install --save tslib@1.6.1
> ```

I think it would be fair by now to just require `"typescript": "^2.3.3"`.